### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by axel7083**
Fixes https://github.com/projectatomic/studio-extension/issues/125

### CODE REVIEW

**Comment by benoitf** [comment]
you can try using `[0.0.202401241015-649826f](https://www.npmjs.com/package/@podman-desktop/api/v/0.0.202401241015-649826f)` waiting 1.7.0 is tagged

File: packages/backend/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "simple-git": "^3.22.0"
   },
   "devDependencies": {
-    "@podman-desktop/api": "0.0.202401191125-9c1aea6",
+    "@podman-desktop/api": "next",
```
Comment (axel7083) [inline packages/backend/package.json:44]:
```suggestion
    "@podman-desktop/api": "0.0.202401241015-649826f",
```

**Review by benoitf — APPROVED** [review]
if you wait ~ 30mn you'll be able to use `1.7.0`

**Comment by axel7083** [comment]
> if you wait ~ 30mn you'll be able to use `1.7.0`

I can wait :) !

File: packages/backend/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "simple-git": "^3.22.0"
   },
   "devDependencies": {
-    "@podman-desktop/api": "0.0.202401191125-9c1aea6",
+    "@podman-desktop/api": "0.0.202401241015-649826f",
```
Comment (benoitf) [inline packages/backend/package.json:44]:
```suggestion
    "@podman-desktop/api": "1.7.0",
```

Comment (benoitf) [reply packages/backend/package.json:44]: it'll also require a yarn.lock file update

**Review by benoitf — APPROVED** [review]

**Review by feloy — APPROVED** [review]

**Pull request merged**